MoSDeF-Installation/README.md

```diff
@@ -0,0 +1,55 @@
+# Instructions to set run MoSDeF Workshops and Tutorials Locally
+Click the following links to take you to installation instructions equivalent to your personal familiarity/expertise with setting up python environments
+
+1. I have conda and python set up, and know how to use conda to install into a local environment -- [expert](#expert-level)
+2. I need to check if I have conda installed -- [double check conda](#check-for-conda)
+3. I need to install conda and have a MAC or LINUX OS -- [mac/os beginner](set_up_your_computers_MacOS.md)
+4. I need to install conda and have a Windows OS -- [windows beginner](set_up_your_computers_Windows.md)
+5. [I'm not certain](#additional-resources) 
+
+## Expert Level
+All python packages are available via conda-forge. Feel free to use `conda install mbuild gmso foyer` to install particular packages to your local environment. You can also view other dependencies in the [environment.yml](../environment.yml).
+
+#### With [mamba](https://github.com/mamba-org/mamba) installed (much faster installation)
+```bash 
+$ git clone https://github.com/mosdef-hub/CECAM-MoSDeF-Workshop.git
+$ cd CECAM-MoSDeF-Workshop
+$ mamba env create -f environment.yml
+$ conda activate mosdef_cecam
+```
+OR
+
+#### Without mamba installed
+```bash 
+$ git clone https://github.com/mosdef-hub/CECAM-MoSDeF-Workshop.git
+$ cd CECAM-MoSDeF-Workshop
+$ conda env create -f environment.yml
+$ conda activate mosdef_cecam
```

Comment:
For the non-mamba installed, I think we still need to point user to use the libmamba solver (https://github.com/conda/conda-libmamba-solver). I think we have issue using just conda with the default solver.

Reply:
What's the issue with just default conda? I swear I've done it recently and it worked okay. I guess I just wanted one version to use mamba since it's fast, or also just have basic conda instructions if people are uncomfortable to switch to something else. That's why both sets of instructions are there, but I recommend the mamba one. 

If for some reason we also need libmamba solver to be installed, then I'd say we're better off just keeping it with one section and requiring users to install mamba in one line as well.

Reply:
it's not just the speed though, the default conda solver sometimes have issue resolving all the version pinning/conflicts, and would end out error out (after taking forever)

Reply:
then let's just do the single installation via Mamba no? Is there a reason to have both a mamba install version and a libmamba install?

Reply:
keeping the `libmamba` instruction might be better. Since I think there will be more folks with `conda`, and if they have `mamba` install, they probably know how to use it.

Reply:
You want to just have the instructions look like:
```bash 
$ git clone https://github.com/mosdef-hub/CECAM-MoSDeF-Workshop.git
$ cd CECAM-MoSDeF-Workshop
$ conda install -n base conda-libmamba-solver
$ conda env create -f environment.yml --solver=libmamba
$ conda activate mosdef_cecam
```

as opposed to:
```bash 
$ git clone https://github.com/mosdef-hub/CECAM-MoSDeF-Workshop.git
$ cd CECAM-MoSDeF-Workshop
$ conda install -c conda-forge mamba
$ mamba env create -f environment.yml
$ conda activate mosdef_cecam
```

Reply:
that's probably the easiest, then we only to have one set of instruction to install `conda` and don't have to worry about `mamba`. I am kinda worry switching between `mamba` and `conda` (even `mamba` can act as `conda` but not the other way around) could be confusing to some.

Reply:
I think I gonna put in the change and merge in the PR. We can decide if some needed to be removed/modified later